src/taco-remote-lib/ios/ios.ts

```diff
@@ -230,14 +229,18 @@ class IOSAgent implements ITargetPlatform {
     }
 
     public debugBuild(buildInfo: utils.BuildInfo, req: Express.Request, res: Express.Response): void {
-        if (this.webProxyInstance) {
-            this.webProxyInstance.kill();
-            this.webProxyInstance = null;
+        if (!global.tacoRemoteLib) {
```

Comment:
I feel it would be nice to have a single place where we manage all the globals. Does it make sense to extract the globals management code to a different class?